### PR TITLE
Add hover-help icon to geo page

### PR DESCRIPTION
**Pull request by ababaian**
I moved the text at the top of the page to a help icon with hover-text and the box-select information to the bottom. This cleans up the page a little bit. Also learning/playing with deployment.

Preview Link: [https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/geo](https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/geo)

### CODE REVIEW

**Comment by victorlin** [comment]
Ah I realize this was your initial ask in the PR, but I got lazy and just dumped all the text directly 🙃

I don't like the hover title, it's not intuitive and takes a second to appear depending on browser. Also this seems like too much text for a title. This is what I'd do as a user:

1. see the `(?)`
2. try clicking it for an action (the icon is a link on other parts of the website)
3. move on without any knowledge since there is no tooltip post-click

If clutter is an issue, I'd recommend an expandable `<details>` section like this but with some styling: https://codesandbox.io/s/2xyryvynqj

P.S. nice seeing you play with website stuff 👍

**Comment by ababaian** [comment]
Yep that's much better. I'll wrestle it in :D

**Comment by ababaian** [comment]
How about this?

 https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/geo

**Comment by victorlin** [comment]
I'm getting some weird behavior, looks like a React issue. try this? https://github.com/facebook/react/issues/15486#issuecomment-488028431

File: src/components/Geo/index.jsx [hunk 1]
```diff
@@ -15,20 +39,24 @@ const Geo = () => {
...
-        </p>
-        <p className="py-1">
-            Use the <b>Box Select</b> or <b>Lasso Select</b> icons in the top-right to retrieve a detailed sample list below the map.
-        </p>
-        <p className="py-1">
-            Currently in Beta Version. Feedback is welcome.
-        </p>
+
+
+        <div><Details title={helpIcon} defaultOpen={false}>
```
Comment (victorlin) [inline src/components/Geo/index.jsx:44]: icons

This seems a bit weird - too many icons. Suggestion: `> Details` or `> Info`

File: src/components/Geo/index.jsx [hunk 2]
```diff
@@ -15,20 +39,24 @@ const Geo = () => {
...
+        <p>This map shows the location of BioSamples from which an intact RdRP sequence could be recovered and geographical meta-data was present.</p>
+        
+        <p> A 100-meter randomization is applied to all points to prevent overplotting.</p>
+        </Details></div>
+
         <div className="my-2">
             <MapPlot setSelectedPoints={setSelectedPoints} />
         </div>
+
+        <div class="text-left text-gray-500">Use the `Box Select` or `Lasso Select` icons in the top-right to retrieve a list of sample details below the map.</div>
```
Comment (victorlin) [inline src/components/Geo/index.jsx:56]: - `text-gray-500` seems too light. At least `600`?
- no HTML styling `<b>Box Select</b>`?

**Review by victorlin — CHANGES_REQUESTED** [review]

File: src/components/Geo/index.jsx (hunk 2 above)
Comment (ababaian) [reply src/components/Geo/index.jsx:56]: fixed

File: src/components/Geo/index.jsx (hunk 1 above)
Comment (ababaian) [reply src/components/Geo/index.jsx:44]: :+1:

**Comment by ababaian** [comment]
How about this? https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/

**Comment by victorlin** [comment]
Styling looks good! But still gets "stuck" after a couple clicks - see my earlier comment https://github.com/serratus-bio/serratus.io/pull/122#issuecomment-799036009

**Comment by ababaian** [comment]
Can't reproduce, not sure what's up.

**Comment by ababaian** [comment]
I changed to a different toggle button with some simpler code and I think this will work better.

https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/

**Comment by ababaian** [comment]
Ignore that last link, with the `css` update the button now works. `main.css` is in gitignore :) 

https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/

File: src/components/Geo/index.jsx [hunk 3]
```diff
@@ -1,10 +1,36 @@
 import React from 'react'
+import ReactDOM from "react-dom";
 import { Helmet } from 'react-helmet';
 import MapPlot from './MapPlot';
 import SelectionInfo from './SelectionInfo';
+import {
+    ExternalLink,
+    helpIcon
+} from '../../common/Helpers';
+import styles from '../../styles/collapse.css';
```
Comment (victorlin) [inline src/components/Geo/index.jsx:10]: styles should go in [tailwind.css](https://github.com/serratus-bio/serratus.io/blob/a862c4d14a4b81d1a760ef383226dfd38c3c9833/src/styles/tailwind.css), this way you won’t need the import here.

File: src/components/Geo/index.jsx [hunk 4]
```diff
@@ -1,10 +1,36 @@
...
 
-const Geo = () => {
+function Details({ defaultOpen = false, title, children }) {
```
Comment (victorlin) [inline src/components/Geo/index.jsx:12]: `Details` is unused, should be removed

File: src/components/Geo/index.jsx [hunk 5]
```diff
@@ -1,10 +1,36 @@
...
+      onKeyDown={handleKey}
+      onClick={() => setOpen(o => !o)}
+    >
+      <summary>{title}</summary>
+      {open && children}
+    </details>
+  );
+}
+
+const Geo = ({ collapsed, children }) => {
```
Comment (victorlin) [inline src/components/Geo/index.jsx:30]: `{ collapsed, children }` denote “props” that would be defined by the parent (in this case [App.jsx](https://github.com/serratus-bio/serratus.io/blob/a862c4d14a4b81d1a760ef383226dfd38c3c9833/src/App.jsx#L38)). They aren’t used and should be removed.

File: src/components/Geo/index.jsx [hunk 6]
```diff
@@ -1,10 +1,36 @@
...
     const [selectedPoints, setSelectedPoints] = React.useState();
+    const [isCollapsed, setIsCollapsed] = React.useState(collapsed);
```
Comment (victorlin) [inline src/components/Geo/index.jsx:32]: `useState(false)` instead of `useState(collapsed)`

**Review by victorlin — CHANGES_REQUESTED** [review]

**Comment by ababaian** [comment]
Let me know :D https://gui-tweak.d1w6d75be7tofa.amplifyapp.com/

**Review by victorlin — APPROVED** [review]
looks great 👍

**Pull request merged**